applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -188,10 +192,12 @@ static int do_secure_socket_open(int peer_verify)
 
 	sec_tag_t sec_tag_list[1] = { sock.sec_tag };
 #if defined(CONFIG_SLM_NATIVE_TLS)
-	ret = slm_tls_loadcrdl(sock.sec_tag);
-	if (ret < 0) {
-		LOG_ERR("Fail to load credential: %d", ret);
-		goto error_exit;
+	if (sock.type == SOCK_STREAM) {
```

Comment:
Why not the same treatment to `SOCK_DGRAM` sockets?

Reply:
Okay. Then please specify what is being fixed or what not in your commit message, that would help to know what to look for (and what not).